src/elements/toggle-switch/src/index.tsx

```diff
@@ -5,63 +5,62 @@ import { jsx } from 'theme-ui'
 interface Props {
   checked: boolean
   onChange?: (e: React.SyntheticEvent) => void
-}
-
-const ToggleSwitch: React.FC<Props> = ({ checked, onChange }) => {
-  const outerStyling = {
-    checked: {
-      backgroundColor: 'black',
-      variant: 'toggleSwitch.outer.variant.checked'
-    },
-    unchecked: {
-      backgroundColor: 'grey', // default value, override in theme
-      variant: 'toggleSwitch.outer.variant.unchecked'
-    }
-  }
-
-  const innerStyling = {
-    checked: {
-      transform: 'translateX(40px)',
-      variant: 'toggleSwitch.inner.variant.checked'
-    },
-    unchecked: {
-      variant: 'toggleSwitch.inner.variant.unchecked'
-    }
+  icons?: {
+    checked: any
+    unchecked: any
   }
+}
 
-  return (
-    <div>
-      <input
-        type="checkbox"
-        className="sr-only"
-        checked={checked}
-        onChange={onChange}
-      />
+const ToggleSwitch: React.FC<Props> = ({ checked, onChange, icons }) => (
+  <div>
+    <input
+      type="checkbox"
+      checked={checked}
+      onChange={onChange}
+      sx={{
+        position: 'absolute',
+        width: 1,
+        height: 1,
+        padding: 0,
+        margin: -1,
+        overflow: 'hidden',
+        clip: 'rect(0, 0, 0, 0)',
+        border: 0
```

Comment:
I think if we're not happy with a CSS rule this should be in a utils package or something to stop it being duplicated every time we need it

Reply:
Yeah I agree, I'm not sure what the best approach is for this as I don't want any users of the component to need to know that they have to add an `sr-only` class. I've moved the css into the theme to abstract the data. Not sure this is the best approach, but in lieu of a utility function it's cleaner than duplicating everywhere - let me know your thoughts

Reply:
seems to be a reasonable approach 👍